RecoLocalTracker/SiPixelClusterizer/plugins/PixelThresholdClusterizer.cc

```diff
@@ -351,6 +358,26 @@ int PixelThresholdClusterizer::calibrate(int adc, int col, int row)
 	  } else {
 	    electrons = int( vcal * theConversionFactor + theOffset); 
 	  }
+
+
+	  // if(adc<0 || adc>255) cout<<" adc wrong range "<<adc<<endl;
+
+	  // if(layer_>0) {
+	  //   if( (layer_==1  && (pedestal<-40. || pedestal>170.) ) || 
+	  // 	(layer_==2  && (pedestal<-260.|| pedestal>230.) ) || 
+	  // 	(layer_==3  && (pedestal<-130.|| pedestal>80.) ) || 
+	  // 	(layer_==4  && (pedestal<-90. || pedestal>110.) ) ) 
+
+	  //     cout<<"PED outside range: layer "<<layer_<<" "<<pedestal<<" "<<DBgain<<" "<<DBpedestal
+	  // 	  <<" "<<adc<<" "<<vcal<<" "<<electrons<<" det: "
+	  // 	  <<detid_<<" row: "<<row<<" col: "<<col<<endl;
+
+	  // }
+
+	  if(electrons<0) 
+	    cout<<" layer "<<layer_<<" "<<adc<<" "<<DBgain<<" "<<pedestal<<" "<<DBpedestal<<" "<<vcal
```

Comment:
cout is not thread safe
I suspect anyhow that also this line was originally supposed to get commented out